bls/bls_linux.go

```diff
@@ -0,0 +1,111 @@
+// +build linux,arm64 !android,linux,amd64,!musl linux,arm,!arm7 arm7 !android,linux,386,!musl !android,musl linux,mips linux,mips64 linux,mips64le linux,mipsle
```

Comment:
Did you generate these from a chart or anything? I'm not confident in my ability to eyeball it for correctness.

Reply:
This is a generated file that is the concatenated list of all the platforms in a package, meaning the `buildDirective`s of all the items under a single key in `platforms.json`. It's generated by `distribute.go`.